xapian-letor/tests/api_letor.cc

```diff
@@ -425,7 +389,7 @@ DEFINE_TESTCASE(listnet_ranker, generated)
     Xapian::ListNETRanker ranker;
     TEST_EXCEPTION(Xapian::FileNotFoundError, ranker.train_model(""));
     string db_path = get_database_path("apitest_listnet_ranker",
-				      db_index_two_documents);
+				       db_index_two_documents);
```

Comment:
The 'd' comes under " now, github has it not aligned properly. This issue was there at a lot of places